### PR TITLE
advance Akka and related SHAs

### PR DESCRIPTION
**Pull request by SethTisue**
Akka 2.5.27 just came out, seems like an opportune time to
bumpity bump

### CODE REVIEW

**Comment by SethTisue** [comment]
queued: ~~https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/5296/~~ https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/5298/

**Comment by SethTisue** [comment]
`FAILURES (UNEXPECTED): akka-http-cors,akka-http-json,lagom,akka-persistence-jdbc`

* akka-http-cors looks like a ScalaTest 3.1 upgrade
* akka-http-json: `de.heikoseeberger.akkahttpjson4s.Json4sSupportSpec`, perhaps an overly sensitive test?
* akka-persistence-jdbc has a failure in the `docs` subproject, so we just need to exclude that (but, see next bullet, we may end up freezing instead in order to keep lagom green?)
* lagom has `Missing dependency: the library com.github.dnvriend#akka-persistence-jdbc is not provided` (perhaps this is somehow the same as the trouble we're having over at #1008, /cc @dwijnand)

**Comment by dwijnand** [comment]
No, it's different in #1008.

akka-persistence-jdbc was recently transfered from https://github.com/dnvriend/akka-persistence-jdbc to https://github.com/akka/akka-persistence-jdbc, which includes changing groupId without changing package name (unfortunately the original artifact, under the com.github.dnvriend groupId, was using the akka package namespace...  So the Akka Team decided to retain the class names and stomach the transition...)

**Comment by SethTisue** [comment]
* akka-http-cors: froze it before ScalaTest 3.1 upgrade
* akka-http-json: we are forked, I updated our fork with `git merge v3.6.7` (which is the version akka-http-json currently expects) and perhaps that will fix it
* akka-persistence-jdbc: made the org change and excluded the `docs` subproject
* lagom: hopefully will go green thanks to the akka-persistence-jdbc org change?

https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/5304/

**Comment by SethTisue** [comment]
uh, I think I went too fast on that akka-persistence-jdbc thing, and wasn't really paying attention, hoping it was an easy fix. digging...

**Comment by SethTisue** [comment]
well, I'm not really sure what was going on with the lagom/akka-persistence-jdbc thing, and I kind of lost track of the interactions between the different things I tried in the course of my flailing, but in any case, it's green now

File: proj/akka-persistence-jdbc.conf [hunk 1]
```diff
@@ -1,9 +1,9 @@
-// https://github.com/dnvriend/akka-persistence-jdbc.git#master
+// https://github.com/dnvriend/akka-persistence-jdbc.git#v3.5.2
```
Comment (ennru) [inline proj/akka-persistence-jdbc.conf:1]: @SethTisue Are you aware that it moved to akka/akka-persistence-jdbc and that's why we changed organisation id for the next major release?

Comment (dwijnand) [reply proj/akka-persistence-jdbc.conf:1]: https://github.com/scala/community-build/pull/1016#issuecomment-564458333

Comment (SethTisue) [reply proj/akka-persistence-jdbc.conf:1]: @ennru thanks! I'll switch to the new repo here once I see that downstream repos are expecting the new organization id

**Pull request merged**